common/vm_get_vbs_status.yml

```diff
@@ -0,0 +1,12 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get Windows VM VBS enablement status
+- include_tasks: vm_get_config.yml
+  vars:
+    property_list: ['config.flags.vbsEnabled']
```

Comment:
Would it fail when getting vbsEnabled flag on ESXi 6.5?

Reply:
It will be null, so I add a check on it when 'set_fact'. Thanks.

Reply:
Got it. So it won't report an error when query a non-exist flag, right?

Reply:
Yeah, it does not fail, just return None.